cmd/util/util.go

```diff
@@ -246,3 +247,34 @@ func KillMountProcess() error {
 	}
 	return mountProc.Kill()
 }
+
+func MaybeChownDirRecursiveToMinikubeUser(dir string) error {
+	if os.Getenv("CHANGE_MINIKUBE_NONE_USER") != "" && os.Getenv("SUDO_USER") != "" {
```

Comment:
Do you need this first os.Getenv check? It looks like you only call this if CHANGE_MINIKUBE_NONE_USER is true.